src/crystal/system/system.cr

```diff
@@ -0,0 +1,24 @@
+# :nodoc
+module Crystal
+  # :nodoc
+  module System
+    # :nodoc
+    module System
```

Comment:
IMO these are not "misc" methods, they are "System" (aka local computer system) utility methods for querying a local system's properties.  

My understanding of the structure is that `Crystal::System` is simply a wrapper namespace for platform specific implementations.

`Crystal::System::System` is meant to contain platform specific implementations of the "System" methods.

It's confusing for sure since we're using the word 'System" for two different things. Perhaps a there is a different name that is more suitable?